### DISCUSSION TITLE
Isomorphic (aka SSR)

### DISCUSSION THREAD

**Discussion by Bloomca** [day8/re-frame#692] [Q&A]
Hi!

Thanks for the README.md, it has a unique style. But what I personally missed, it is a list of features. Basically, it looks like a React + Redux (with less boilerplate) packed into one thing. It seems we use one `dispatch`, so server-side rendering is not possible (at least with data population). Is it true?

Thanks!

**Comment by mike-thompson-day8** [comment]
@Bloomca there's some resources on server side rendering at the end of this :
https://github.com/Day8/re-frame/blob/master/docs/External-Resources.md

**Comment by Bloomca** [comment]
Oh, thanks, missed it.

SSR definitely is a buzzword now, but sometimes it delivers better experience, so for me it is an important thing to consider over react/vue.

Thanks again, hope to try your library in some small app (completely tired from huge boilerplate in react).
